jenkins/deploy-clio-server.sh

```diff
@@ -86,46 +80,20 @@ CLIO_APP_CONF=clio.conf
 # name of the logback config used by the clio instance
 CLIO_LOGBACK_CONF=clio-logback.xml
 # port to expose for clio in the clio container
-CONTAINER_CLIO_PORT=8000
+CONTAINER_CLIO_PORT=8080
```

Comment:
We override it everywhere when we deploy, because I missed that we already hard-code exposing 8080 in the SBT-defined Dockerfile. I thought about removing this variable but then we'd have a bunch of magic 8080's floating around in the config templates.